tests/test_infra.py

```diff
@@ -74,3 +77,11 @@ def test_ensure_copyright():
     if issues:
         msg = "Please add copyright headers to the following files:\n" + "\n".join(issues)
         pytest.fail(msg, pytrace=False)
+
+
+def test_setup_version():
+    """Verify that setup.py is picking up the version correctly."""
+    cmd = [os.path.abspath('setup.py'), '--version']
```

Comment:
do you want
cmd = [sys.executable, os.path.abspath('setup.py')...
?

Reply:
setup.py is already executable, so just used it...